mlir-sycl/lib/Conversion/SYCLToGPU/SYCLToGPU.cpp

```diff
@@ -106,7 +98,8 @@ void convertToFullObject(ConversionPatternRewriter &rewriter, StringRef opName,
   const auto loc = op->getLoc();
   const auto targetIndexTy = rewriter.getIntegerType(64);
   const auto getIndexTy = rewriter.getIntegerType(32);
-  const auto dimMtTy = MemRefType::get(dimensions, targetIndexTy, {}, 4);
+  const auto underlyingArrTy =
+      MemRefType::get(dimensions, targetIndexTy, {}, 4);
```

Comment:
Is it possible to not hard code the address space?

Reply:
Done!